### DISCUSSION TITLE
Is it possible to have questions on one long form?

### DISCUSSION THREAD

**Discussion by tony-williams-maxcomputing** [formbricks/formbricks#4248] [General Requests]
Is it possible to have questions on one long form or at least a question then a file upload field?

I am building a form where I need people to upload evidence for their answer.

**Comment by mattinannt** [comment]
At Formbricks we only allow one question per page, but of course you can add as many questions as you like including file upload questions.

<img width="748" alt="Screenshot 2024-11-08 at 12 31 32" src="https://github.com/user-attachments/assets/45f89b77-4ad7-4174-9d15-a07ad6b59be1">
